cypress/e2e/map/map.cy.js

```diff
@@ -1226,8 +1256,12 @@ describe('Map interactions', () => {
         cy.url().should('include', '?polygon[0]=42.1875%2C-76.46517%2C42.1875%2C-82.40647%2C56.25%2C-76.46517%2C42.1875%2C-76.46517&sf=1&sfs[0]=0&lat=-90&projection=EPSG%3A3031&zoom=0')
 
         // draws a polygon on the map
-        cy.get('.leaflet-interactive').first().should('have.attr', 'd', 'M768 358L821 299L850 333L768 358z')
-        cy.get('.leaflet-interactive').last().should('have.attr', 'd', 'M821 299L768 358L850 333L821 299z')
+        // cy.get('.leaflet-interactive').first().should('have.attr', 'd', 'M768 358L821 299L850 333L768 358z')
+        // cy.get('.leaflet-interactive').last().should('have.attr', 'd', 'M821 299L768 358L850 333L821 299z')
```

Comment:
Are these commented out but plan to be commented back in like the other TODOs?

Reply:
Left over from changing the way the assertions worked, removed these